src/app/components/HoldPage/HoldConfirmation.jsx

```diff
@@ -78,7 +78,7 @@ class HoldConfirmation extends React.Component {
                 ]}
               >
                 <TabPanel id="building">
-                  <iframe src={`${location.address["map-embed-uri"]}`} height="450" frameBorder="0" style={{border: 0}} allowFullScreen></iframe>
+                  <iframe src={`${location.address["map-embed-uri"]}`} height="450" frameBorder="0" style={{border: 0}} allowFullScreen title="Google Map" tabIndex="-1"></iframe>
```

Comment:
@beefoo I'm adding a tabIndex of -1 to the iframes so that a user can't tab into those elements. It's something we can address later, but for now a user can get lost in the Hathi Trust or Google Maps viewers.